pkg/config/containers.conf

```diff
@@ -669,6 +669,9 @@ default_sysctls = [
 # A value of 0 is treated as no timeout.
 #volume_plugin_timeout = 5
 
+# Default timeout in seconds for podmansh logins.
+podmansh_timeout = 30
+
```

Comment:
all defaults should be commented out by default,
defaults should be set in the code to ensure it works even without config file installed

Reply:
ack, will update